### ISSUE TITLE
Renaming copy of HTML file also renames original file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bwebb3856/qwickie#53]
```
> What steps will reproduce the problem?
1. Copy an HTML file
2. Paste the HTML file into a different package
3. Rename the new HTML file

> What is the expected output? What do you see instead?
Only the new HTML file should be renamed. Instead, the source HTML file - and 
all of its corresponding files (Java class, properties) are also renamed.

> What version of the product are you using? On what operating system?
QWickie 1.1.4 on Eclipse 4.3.2



```

Original issue reported on code.google.com by `organsny...@gmail.com` on 12 May 2014 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
Ok, see it. Working on it.
Thanks for your report.
```

Original comment by `count.negative` on 12 May 2014 at 10:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
could you please take a look at 1.1.5?
```

Original comment by `count.negative` on 16 May 2014 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
Looks fixed to me! I've also upgraded to Eclipse 4.4-M7 since I filed the 
report, and I'm happy to report that QWickie seems to be working well.

Thanks so much for the great tool!
```

Original comment by `organsny...@gmail.com` on 18 May 2014 at 9:49

**Comment by GoogleCodeExporter** [comment]
```
Ok, so thank you again for your report and let me know if there's something in 
4.4 (not tried yet)!
```

Original comment by `count.negative` on 18 May 2014 at 10:47
- Changed state: **Fixed**
